kodilla-patterns2/src/main/java/com/kodilla/patterns2/observer/homework/Homework.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.patterns2.observer.homework;
+
+import java.util.ArrayDeque;
+import java.util.ArrayList;
+import java.util.Deque;
+import java.util.List;
+
+public class Homework implements Observable {
+
+    private Deque<String> homeworks = new ArrayDeque<>();
+    private List<Observer> teachers = new ArrayList<>();
+    private String studentName;
```

Comment:
Dlaczego to sie nazywa Homework, nie Student? "Student" bedzie mialo wiecej sensu